Gopkg.lock

```diff
@@ -9,15 +9,9 @@
 
 [[projects]]
   branch = "master"
-  name = "github.com/Azure/go-ansiterm"
```

Comment:
ah, didn't spot the commit split - nice move.
re: is it a bug - well, I guess that's up to the dep authors, but seeing as dep was given a single explicit instruction (update this one package), and it wen't and did some other (admittedly not 'incorrect breakage') stuff - really I'd like them to have an option so you have to knowingly 'force' the updates in this case or something. If I find a minute I'll trawl the dep code/github to see if they've covered this before etc.

lgtm